components/org.wso2.carbon.identity.api.expired.password.identification/org.wso2.carbon.identity.api.expired.password.identification.v1/src/main/resources/ExpiredPasswordIdentification.yaml

```diff
@@ -26,7 +26,10 @@ paths:
     get:
       tags:
         - Get password expired users
-      description: Get password expired users list for a specified period.
+      description: |
```

Comment:
This should affect to generated files. Let's add the generated file changes as well

Reply:
Updated